### ISSUE TITLE
reloading a search result page triggers 500

### ISSUE THREAD

**Issue by azul** [helpyio/helpy#595]
In the admins panel, if i type in a search string and search for it i get a fine set of results. But once i reload that page I get a 500 error.

I guess this controller only responds to json.
```
ActionView::MissingTemplate (Missing template admin/search/search with {:locale=>[:en], :formats=>[:html], :variants=>[], :handlers=>[:erb, :builder, :raw, :ruby, :haml, :jbuilder]}. Searched in:
  * "/srv/helpy/app/views"
  * "/srv/helpy/vendor/bundle/ruby/2.3.0/gems/devise_invitable-1.6.0/app/views"
  * "/srv/helpy/vendor/bundle/ruby/2.3.0/gems/twitter-bootstrap-rails-3.2.2/app/views"
  * "/srv/helpy/vendor/bundle/ruby/2.3.0/gems/kaminari-0.16.3/app/views"
  * "/srv/helpy/vendor/bundle/ruby/2.3.0/gems/grape-swagger-rails-0.3.0/app/views"
  * "/srv/helpy/vendor/bundle/ruby/2.3.0/gems/devise-bootstrap-views-0.0.8/app/views"
  * "/srv/helpy/vendor/bundle/ruby/2.3.0/gems/devise-i18n-1.0.1/app/views"
  * "/srv/helpy/vendor/bundle/ruby/2.3.0/gems/devise-3.5.6/app/views"
):
  app/controllers/admin/search_controller.rb:39:in `topic_search'
  app/controllers/application_controller.rb:206:in `set_time_zone'
```

**Comment by scott** [comment]
I was unable to recreate this in testing.  Reload seems to work correctly.

**Comment by yasirazgar** [comment]
I too tried this in my local development env, it seems to work fine

**Comment by azul** [comment]
I'm upgrading to 1.2.1 to see if i can still reproduce this.

**Comment by azul** [comment]
Seems fixed with 1.2.1. Thanks a lot!
